lib/Pipfile

```diff
@@ -11,5 +11,6 @@ pyyaml = "*"
 # TODO: Update Dockerfile to use newer libgit2 version
 pygit2 = "==1.5.0"
 attrs = "*"
+redis = "*"
```

Comment:
See comment below on keeping the redis cache implementation in the worker instead.